### ISSUE TITLE
Facebook Lead Ads

### ISSUE THREAD

**Issue by tokumotion** [daroczig/fbRads#35]
Facebook developed a new ad format. It creates leads from fb users, can the package read those leads?

**Comment by daroczig** [comment]
Yeah, it should be pretty trivial to add support for this based on https://developers.facebook.com/docs/marketing-api/guides/lead-ads/leads

PR welcome, but I will also try to work on this -- around mid/2nd half of October.

**Comment by tokumotion** [comment]
Great! Thank you!

**Comment by prasoonsrivas** [comment]
Hi All 

@daroczig that is what I was asking you from last week.  Facebook Lead Generation... 
@tokumotion you can,  yes the package can read those leads ..  , But it gives a messy list..

**Comment by tokumotion** [comment]
@prasoonsrivas I got it to work, but the data Facebook serves back is a list of leads.

I got the data into a data.frame by using this code:

```
leads <- do.call('rbind', lapply(list, data.frame))
```

That's pretty much it.

**Comment by ArbenKqiku** [comment]
Hi Everyone,

Could you share your code to retrieve the leads from Facebook? That would be great, thank you :)

**Comment by ArbenKqiku** [comment]
> @prasoonsrivas I got it to work, but the data Facebook serves back is a list of leads.
> 
> I got the data into a data.frame by using this code:
> 
> ```
> leads <- do.call('rbind', lapply(list, data.frame))
> ```
> 
> That's pretty much it.

Hi @tokumotion, could you share your code to retrieve lead ads with R? I am really struggling with it, that would be helpful :) Thank you!
